Sources/Sentry/Public/SentryDefines.h

```diff
@@ -171,16 +171,6 @@ typedef NSNumber *_Nullable (^SentryTracesSamplerCallback)(
 typedef void (^SentrySpanCallback)(id<SentrySpan> _Nullable span DEPRECATED_MSG_ATTRIBUTE(
     "See `SentryScope.useSpan` for reasoning of deprecation."));
 
-/**
- * Log level.
- */
-typedef NS_ENUM(NSInteger, SentryLogLevel) {
```

Comment:
This was replaced by `SentryLevel` if i understood correctly. I removed this for now to avoid a naming conflict, but I guess we still want to keep this around?

Reply:
While I agree it is unused, we should leave it in because it is a public header.
We could consider marking it as deprecated and leave a comment in [the v9 discussion](https://github.com/getsentry/sentry-cocoa/discussions/2946)